frontend/ui/data/css_tweaks.lua

```diff
@@ -1031,6 +1031,46 @@ This tweak can be duplicated as a user style tweak when books contain footnotes
             ]],
             separator = true,
         },
+        {
+            id = "following-footnote-inpage_cr_only_if",
+            conflicts_with = function(id) return util.stringStartsWith(id, "following-footnote-inpage_") end,
+            title = _("Extend in-page footnotes until next footnote"),
+            description = _([[
+Show blocks following footnotes as part of the footnote.]]),
+            css = [[
+*, autoBoxing {
+    -cr-hint: late;
+    -cr-only-if: following-inpage-footnote -inpage-footnote;
+    -cr-hint: extend-footnote-inpage;
+    margin: 0 !important;
```

Comment:
More indentation on the lines following a `-cr-only-if:` (to make it visually clear the next apply only-if :) ala python.
cf the existing ones.